neomodel/__init__.py

```diff
@@ -18,4 +18,4 @@
 __email__ = 'robin.ge@gmail.com'
 __license__ = 'MIT'
 __package__ = 'neomodel'
-__version__ = '3.2.4'
+__version__ = '3.2.6'
```

Comment:
it'd be actually preferable to `= pkg_resources.get_distribution('neomodel').version`.

Reply:
That's a much better idea!